### PR TITLE
fix doc for ISO date input

### PR DESCRIPTION
**Pull request by leesei**
original code contains error

 Changes to be committed:
	modified:   docs/Inputs.md
	modified:   docs/_layouts/default.html

### CODE REVIEW

**Comment by fzaninotto** [comment]
There is no mention of the store anywhere in the doc, on purpose. At a basic level, a developer shouldn't need to know about Flux to use aor. Can you remove the comments in the jsx and the section name change?

**Comment by leesei** [comment]
What if I change the wording from store to record?
AOR does store the value somewhere and sent that to the API endpoint.
It seems the term record has been used in the docs.

**Comment by fzaninotto** [comment]
I'm fine with that.

File: docs/Inputs.md [hunk 1]
```diff
@@ -940,38 +940,40 @@ You can choose a specific input type using the `type` attribute, for instance `t
 
 **Warning**: Do not use `type="number"`, or you'll receive a string as value (this is a [known React bug](https://github.com/facebook/react/issues/1425)). Instead, use [`<NumberInput>`](#numberinput).
 
-## Transforming Input Value to/from Store
+## Transforming Input Value to/from Record
 
-The data format returned by the input component may not be what your store desires. Since Admin-on-rest uses Redux Form, we can use its `parse()` and `format()` functions to transform the input value to and from the store. It's better to understand the [input value's lifecycle](http://redux-form.com/6.5.0/docs/ValueLifecycle.md/) before you start.
+The data format returned by the input component may not be what your record desires. Since Admin-on-rest uses Redux Form, we can use its `parse()` and `format()` functions to transform the input value to and from the record. It's better to understand the [input value's lifecycle](http://redux-form.com/6.5.0/docs/ValueLifecycle.md/) before you start.
```
Comment (fzaninotto) [inline docs/Inputs.md]: Does not make sense anymore

File: docs/Inputs.md [hunk 2]
```diff
@@ -940,38 +940,40 @@ You can choose a specific input type using the `type` attribute, for instance `t
...
 
 ```jsx
 <NumberInput source="percent" format={v => v*100} parse={v => v/100} label="Formatted number" />
 ```
 
-`<DateInput>` stores and returns a `Date` object. If you would like to store the ISO date `"YYYY-MM-DD"` in your store:
+`<DateInput>` stores and returns a `Date` object. If you would like to store the ISO date `"YYYY-MM-DD"` in your record:
 
 ```jsx
+// record -> input
```
Comment (fzaninotto) [inline docs/Inputs.md]: In fact, no need to add this comment since the mnemonic was given previously. Same for line 970.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**